src/mlpack/tests/metric_test.cpp

```diff
@@ -87,4 +97,38 @@ BOOST_AUTO_TEST_CASE(LINFMetricTest)
                       lMetric.Evaluate(a2, b2), 1e-5);
 }
 
+/**
+ * Simple test for IoU metric.
+ */
+BOOST_AUTO_TEST_CASE(IoUMetricTest)
+{
+  arma::vec bbox1(4), bbox2(4);
+  bbox1 << 1 << 2 << 100 << 200;
+  bbox2 << 1 << 2 << 100 << 200;
+  // IoU of same bounding boxes equals 1.0.
+  BOOST_REQUIRE_CLOSE(1.0, IoU<>::Evaluate(bbox1, bbox2), 1e-4);
+
+  // Use coordinate system to represent bounding boxes.
+  // Bounding boxes represent {x0, y0, x1, y1}.
+  bbox1 << 39 << 63 << 203 << 112;
+  bbox2 << 54 << 66 << 198 << 114;
+  // Value calculated using python interpreter.
```

Comment:
Let's change `python` to `Python` :)
```suggestion
  // Value calculated using Python interpreter.
```

Reply:
Awesome!, Done. Thanks a lot for the reviews.